ggml/src/CMakeLists.txt

```diff
@@ -671,7 +685,10 @@ if (GGML_KOMPUTE)
                 ${CMAKE_CURRENT_SOURCE_DIR}/kompute-shaders/op_getrows.comp
                 ${CMAKE_CURRENT_SOURCE_DIR}/kompute-shaders/op_mul_mv_q_n_pre.comp
                 ${CMAKE_CURRENT_SOURCE_DIR}/kompute-shaders/op_mul_mv_q_n.comp
-                COMMAND ${glslc_executable} --target-env=vulkan1.2 -o ${spv_file} ${CMAKE_CURRENT_SOURCE_DIR}/${source}
+                ${GGML_HEADER_PATH}
+                COMMAND ${glslc_executable} --target-env=vulkan1.2
+                        -o ${spv_file} ${CMAKE_CURRENT_SOURCE_DIR}/${source}
+                        -DGGML_ROPE_TYPE_NEOX=${GGML_ROPE_TYPE_NEOX_VALUE}
```

Comment:
No need for the kompute-related changes in the CMake build system. Simply define the `GGML_ROPE_TYPE_NEOX` constant in the kompute shaders where needed for now

Reply:
Ah great, I've added this to `rope_common.comp` in https://github.com/ggerganov/llama.cpp/pull/8949/commits/cfb98641828d79d7ffba94f74b970aed133a520b.